.github/workflows/integration-test.yml

```diff
@@ -40,7 +40,9 @@ jobs:
 
       - name: Collect docker logs on failure
         if: failure()
-        uses: jwalton/gh-docker-logs@v2
+        # pinned jwalton/gh-docker-logs@v2.2.1 to its commit sha
+        # https://github.com/jwalton/gh-docker-logs/releases/tag/v2.2.1
+        uses: jwalton/gh-docker-logs@59c9656cd3cb7542525f3dce7ae2f44c0ff85d66
```

Comment:
In the PR you updated this at first, you've updated also `actions/checkout`, `actions/setup-node`, and `actions/upload-artifact`.

Should we do the same pin to all of them?